### ISSUE TITLE
[IDEA] Support for a "Blocked" status

### ISSUE THREAD

**Issue by trafficone** [trafficone/noscrum#30]
**Will this feature solve a problem for you? Please describe it.**
Sometimes a task cannot be accomplished because of something outside of my control, it would be nice to be able to register that state in NoScrum.

**Describe the solution you'd like**
MVP: A new status of "Blocked" for Tasks
Stretch Goal 1: A Blocked task can have a note indicating how to unblock it
Stretch Goal 2: A blocked task can have an associated task which details how to unblock it.

**Describe alternatives you've considered**
- Overloading "In-Progress" status -> does not make it clear that work is not being done.
- Changing task name -> makes it difficult to track work over time
- Creating "task blocker" tasks -> unnecessarily clutters work with non-tasks
